src/components/Inline-quotes/index.js

```diff
@@ -50,6 +51,12 @@ text-align: center;
     @media screen and (max-width: 500px) {
         text-align: center;
     }
+    img{
+        border-radius: 50%;
+        width: 90px;
+        height: 90px;
```

Comment:
@Saksham294 let's make this responsive with use of max-height, %s, or some other dynamic CSS approach.